src/containers/GridContainer.js

```diff
@@ -13,7 +13,7 @@ function mapStateToProps(state) {
 function mapDispatchToProps(dispatch) {
   return {
     changeGrid: gridId => dispatch(changeGrid(gridId)),
-    resetGrid: () => dispatch(resetGrid()),
+    resetGrid: resetGrid,
```

Comment:
if we do need to dispatch `resetGrid`, this wouldn't operate as we expect. we can convert this whole thing to object notation